decomp/semantics/uds.py

```diff
@@ -125,18 +145,31 @@ def __init__(self,
                             spl.add_annotation(ann)
 
                         if sname == split or split is None:
-                            json_name = 'uds-ewt-'+sname+'.json'
-                            json_path = os.path.join(self.__class__.DATA_DIR,
+                            # Prepare graphs
+                            graphs_json_name = 'uds-ewt-sentences-'+sname+'.json'
```

Comment:
See my comment above on munging filenames. I think we should explicitly represent this structure as part of the file system and use utilities like those in `os.path`.

Reply:
Will do.